README.md

```diff
@@ -1,2 +1,83 @@
 [![Work in Repl.it](https://classroom.github.com/assets/work-in-replit-14baed9a392b3a25080506f3b7b6d57f295ec2978f6f33ec97e36a161684cbe9.svg)](https://classroom.github.com/online_ide?assignment_repo_id=4674825&assignment_repo_type=AssignmentRepo)
 # Careers in Computer Science
+## Description
+- Web Developer 
```

Comment:
The actual job you've based your assignment on should be given greater emphasis then just a bullet point